sources/packages/backend/libs/services/src/sfas/sfas-individual.service.ts

```diff
@@ -38,4 +38,34 @@ export class SFASIndividualService extends DataModelService<SFASIndividual> {
 
     return individual?.pdStatus;
   }
+
+  /**
+   * Search for a record in SFAS table using student details.
+   * @param lastName student's last name.
+   * @param birthDate student's birth date.
+   * @param sin student's sin.
+   * @returns SFAS individual details.
+   */
+  async getSFASOverawards(
+    lastName: string,
+    birthDate: string,
+    sin: string,
+  ): Promise<SFASIndividual> {
```

Comment:
Just for my understanding. I assume there will be only one record per student in sfas_individuals right?

Reply:
Yes @dheepak-aot, that is the assumption. At least the combination of sin/last name/DOB should be unique.

Reply:
Yes, that is what we expect from SFAS.